app/pipeline/chat/course_chat_pipeline.py

```
@@ -11,6 +11,7 @@
 from langchain_core.output_parsers import JsonOutputParser
 from langchain_core.prompts import (
     ChatPromptTemplate,
+    SystemMessagePromptTemplate,
```

Ruff (0.8.2)

`langchain_core.prompts.SystemMessagePromptTemplate` imported but unused

Remove unused import: `langchain_core.prompts.SystemMessagePromptTemplate`

(F401)